chart/skywalking/files/conf.d/oap/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
What is this?

Reply:
When testing locally, we usually put the files under this directory(e.g. `application.yml`), but they are not supposed to be checked into the codebase, so we just ignore all(*) of them, while keeping the directory hierarchy